rikaichan.js

```diff
@@ -164,15 +164,15 @@ var rcxMain = {
 				return false;
 			} */
 			try {
-				this.dict = new rcxDict(this.haveNames/* && !this.cfg.nadelay*/);
+				this.dict = new rcxDict();
+				return this.dict.init(this.haveNames/* && !this.cfg.nadelay*/);
 				//this.dict.setConfig(this.dconfig);
 			}
 			catch (ex) {
 				alert('Error loading dictionary: ' + ex);
```

Comment:
This is no longer reachable right?

It seems like instead we'd want to use thenCatch at the call site?